.github/workflows/e2e-k3s.yaml

```diff
@@ -44,17 +50,15 @@ jobs:
         run: |
           kubectl create ns sample-application
           kubectl -n sample-application apply -Rf sample-apps/hotrod/
+          kubectl -n sample-application get deploy --output name | xargs -r -n1 -t kubectl -n sample-application rollout status
```

Comment:
Hotrod app should take less than few minutes to get ready.
Feel free include the `timeout` of `300s`  here. If opt out, it will default to 10 minutes.

Reply:
There is not `--timeout` flag for the rolout command, it was a mistake to have it from the first place.